src/components/design-system/_colors.dev.jsx

```diff
@@ -5,148 +5,148 @@ export default { title: "Design system/Colors" };
 
 export function Bases() {
   return (
-    <div className="ddb-color-example-container">
+    <div className="dpl-color-example-container">
```

Comment:
I got an error when I deleted the `design-system` folder because of [this](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/feature/separat-opdateringtypescript-projekt-klargor-dpl-cms-DDFSOEG-90/postcss.config.js#L14:L14).
But we might as well delete that too?

Reply:
save _system.scss with  commentary